bootstrap/pkg/kfupgrade/kfupgrade.go

```diff
@@ -52,7 +52,7 @@ func createNewKfApp(baseConfig string, version string, oldKfDef *kfdefsv3.KfDef)
 	// Load the new KfDef from the base config
 	newKfDef, err := kfdefsv3.LoadKFDefFromURI(baseConfig)
```

Comment:
Use this function
https://github.com/kubeflow/kubeflow/blob/9f5ef66ce3d12ff2b603d07330a16661caa0b286/bootstrap/pkg/apis/apps/configconverters/converters.go#L37 to load a KFConfig rather than a KFDef